scripts/globals/abilities/pets/mountain_buster.lua

```diff
@@ -22,6 +22,8 @@ function onPetAbility(target, pet, skill)
 	totaldamage = AvatarFinalAdjustments(damage.dmg,pet,skill,target,MOBSKILL_PHYSICAL,MOBPARAM_SLASH,numhits);
 	target:delHP(totaldamage);
 	target:updateEnmityFromDamage(pet,totaldamage);
-
+	
+	local typeEffect = EFFECT_BIND;

```

Comment:
All you did there was create  local variable named "typeEffect" and store the global variable EFFECT_BIND in it. You have no code making use of typeEffect to actually apply EFFECT_BIND to anything.